amxmodx/CGameConfigs.cpp

```diff
@@ -886,6 +887,17 @@ CGameConfigManager::~CGameConfigManager()
 {
 }
 
+void CGameConfigManager::OnAmxxStartup()
+{
+	char error[256]; *error = '\0';
+
+	if (!LoadGameConfigFile("common.games", &CommonConfig, error, sizeof(error)) && *error != '\0')
```

Comment:
Should this really not log *at all* if there is no error message (not that it could happen, but still)?